### PR TITLE
코드 설명용 pr (머지x)

### PR DESCRIPTION
**Pull request by hagiss**
코드 설명용 pr입니다.

### CODE REVIEW

File: fine_tune.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        self.log('val_acc', self.flat_accuracy(logits.detach().cpu().numpy(), label.cpu().numpy()),
+                 prog_bar=True)
+
+
+to_tensor_transform = T.Compose([T.ToTensor()])
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='vit-simclr-fine-tune')
+
+    parser.add_argument('--checkpoint', '-ch', required=True, type=str, help='checkpoint path')
```
Comment (hagiss) [inline fine_tune.py:125]: pl_train.py에서 tensorboard가 저장한 checkpoint path

File: pl_train.py [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+        loss2 = self.criterion(logits2, labels)
+        loss = loss1 + loss2
+        loss /= 2
+        return loss
+
+    def training_step(self, batch, _):
+        aug1 = self.augment1(batch[0])
+        aug2 = self.augment2(batch[0])
+        output_loss = self.forward(torch.cat((aug1, aug2), dim=0))
+        # layer_loss = self.get_layer_loss() / (self.depth - 1)
```
Comment (hagiss) [inline pl_train.py:104]: 주석처리된 코드를 활성화 하면 layer loss 추가됩니다.
ratio는 layer loss를 학습이 진행될수록 더 크게 줄려고 했는데 학습 후반부에 layer loss가 커지면 다시 학습이 저해되는 효과가 발생

**Comment by hagiss** [comment]
pretrain : python3 pl_train.py --name vit
단순히 위처럼 실행하면 됩니다. --name은 tensor board 에 폴더 이름 저장용 입니다.
linear : python3 fine_tune.py -ch [checkpoint path (pl_train.py의 tensorboard logger가 저장한)] --name vit_fine
위와 같은 방식으로 코드 돌리면 됩니다

**Comment by hagiss** [comment]
코드에 대한 질문이 있으시면 여기에 그대로 남겨주시면 될듯합니다

**Comment by hagiss** [comment]
I'm mistaken git origin, I'm sorry
